.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -85,7 +85,7 @@ This is a checklist for PR authors & reviewers. Please make sure to complete all
 <details>
 <summary><h4>PR Reviewer Checklist</h4>
 
-The Contributor+ will copy/paste it into a new comment and complete it after the author checklist is completed
+The reviewer will copy/paste it into a new comment and complete it after the author checklist is completed
```

Comment:
NAB, but I think this is ambiguous still as it implies that there is only one "reviewer". Some PRs have multiple reviewers. Who should do the checklist if no C+ is assigned?

Reply:
I think the answer is that a C+ should always be assigned - but seems like there is not a total consensus on this and we haven't made it a formal process yet.

Reply:
For the C+ review, I think we can update (if required) the C+ doc to state that C+ should always have to fill out the PR reviewer checklist when they're assigned as reviewers.